src/docs/pages/component-docs/components/user-menu/user-menu-doc.component.ts

```diff
@@ -16,29 +16,29 @@ import { UserMenuPlaygroundKnobs } from './examples/playground.component';
         <app-component-doc-scaffold mdFileName="UserMenu.md" [knobGroups]="knobGroups">
             <div examples class="app-example">
                 <div class="example-section">
-                    <div class="example-heading">Basic User Menu</div>
+                    <div class="example-heading">User Menu</div>
                     <div class="example-description">
-                        The <code>&lt;blui-user-menu&gt;</code> accepts an <code>avatar</code> input and a
-                        <code>blui-menu-body</code> overlay content. The <code>open</code> input is used to manage
-                        state.
+                        The <code>&lt;blui-user-menu&gt;</code> is a combination of an avatar and menu that is used to
```

Comment:
Emphasize that this is from ng mat, and put a link to ng's documentation

Reply:
Angular Material doesn't have an Avatar, but I've updated the description to link to the Menu API.